constellations/orion/src/repos/migrations/000022_add_price_lump_at_classes.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE classes
+    ADD COLUMN price_lump INT UNSIGNED;
```

Comment:
This is correct. If a column does NOT have `NOT NULL` or `DEFAULT 0` it means it can be NULL. So we just have to change the go code to support nullable values.